src/server/services/process-upload.js

```diff
@@ -102,7 +101,8 @@ const processUpload = async ({
           docs.push(doc)
         }
       })
-      subrecipients.update(subs)
+      // FIXME: Save subrecipients data to subrecipients table?
+      // subrecipients.update(subs)
```

Comment:
not sure if this is the place to comment this out. are you having problems saving the sub-recipients? are they in a different schema?